frontend/src/components/company-survey-v2/project.tsx

```diff
@@ -46,3 +48,12 @@ export async function getProjectConfiguration(projectName: ProjectName): Promise
         buurtcodes,
     }
 }
+
+export async function getProjectById(projectId: string): Promise<Project | null> {
+    try {
+        return await ztorFetch(`/projects/${projectId}`);
+    } catch (error) {
```

Comment:
I don't think we should catch these errors. Later we can make an error boundary in React to display it nicer.